### PR TITLE
[ADD] delivery_translatable in 15.0

### PR DESCRIPTION
**Pull request by ikapasi-initos**

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
Code review and functional test. LGTM!

**Comment by ikapasi-initos** [comment]
Hello @hparfr it would be nice if you review this pull.

**Comment by ikapasi-initos** [comment]
Hi, @sebalix can you please review this pull..?

File: delivery_translatable/models/delivery.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    name = fields.Char(required=True, translate=True)
+    website_description = fields.Text(
+        string="Description for the website", translate=True
+    )
```
Comment (sebalix) [inline delivery_translatable/models/delivery.py]: Better to move this override in another module in OCA/website to not bring the `website` dependency in this repository.

**Review by sebalix — COMMENTED** [review]
Hi, for such changes, it could be nice to propose this to Odoo directly, at least on `master` to get their feedback?
At least the website description is an important one.

File: delivery_translatable/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Make Delivery Text Properties Translatable",
+    "category": "Hidden",
+    "summary": """name and website_description field of delivery carrier setup translatable.""",
+    "version": "15.0.1.0.0",
+    "depends": [
+        "delivery",
+        "website_sale_delivery",
```
Comment (sebalix) [inline delivery_translatable/__manifest__.py]: Dependency against `website` to remove

File: delivery_translatable/README.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+=====================
+Delivery Translatable
+=====================
+
+* name and website_description field of delivery carrier setup translatable.
```
Comment (sebalix) [inline delivery_translatable/README.rst]: This has to be moved (without the title) to the `readme/` subfolder, in `readme/DESCRIPTION.rst` (check other modules in this repository how it is done, there is also `readme/CONTRIBUTORS.rst` to fill). Then the `README.rst` will be generated automatically by the OCA bot.

EDIT: Here are all the fragments supported in `readme/`: https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py#L29-L39

**Review by sebalix — APPROVED** [review]

**Comment by ikapasi-initos** [comment]
It would be nice maintainer approved this pull.

**Pull request merged**